catalog/dags/common/storage/media.py

```diff
@@ -242,19 +249,19 @@ def _flush_buffer(self) -> int:
         return buffer_length
 
     @staticmethod
-    def _tag_blacklisted(tag: str | dict) -> bool:
+    def _tag_denylisted(tag: str | dict) -> bool:
         """
-        Determine if the is banned or contains a banned substring.
+        Determine if the tag is banned or contains a banned substring.
 
         :param tag: the tag to be verified against the blacklist
         :return: true if tag is blacklisted, else returns false
         """
         if isinstance(tag, dict):  # check if the tag is already enriched
             tag = tag.get("name")
```

Comment:
```suggestion
```

Reply:
I left this untouched precisely to minimize changes 😄 As this part of the code does not affect the rest of the operations, I thought it might be better to avoid it in this instance, but I can remove it if we agree on this path.

Reply:
I thought an easier path to minimizing would be not touching the provider scripts, and handling all of the conversion here, in the `MediaStore` :)